ui/components/app/confirm/info/row/url.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../../component-library';
+import {
+  AlignItems,
+  BorderRadius,
+  Display,
+  FlexWrap,
+  IconColor,
+  TextColor,
+  TextVariant,
+  BackgroundColor,
+} from '../../../../../helpers/constants/design-system';
+
+export type ConfirmInfoRowUrlProps = {
+  url: string;
+};
+
```

Comment:
The name `ConfirmInfoRowUrl` is un-intuitive. We should not couple these components to `ConfirmInfoRow`. These can be independent component which can be wrapped in `ConfirmInfoRow`.

Reply:
note: if we change this, we can do this in a future PR and apply changes to all relevant components